### PR TITLE
Read password from file not containing trailing newline 

### PR DESCRIPTION
**Pull request by fa993**
Addresses #426

### CODE REVIEW

File: src/repository/mod.rs [hunk 1]
```diff
@@ -1,7 +1,7 @@
-use std::fs::File;
 use std::io::BufReader;
 use std::path::PathBuf;
 use std::process::Command;
+use std::{fs::File, io::BufRead};
```
Comment (aawsome) [inline src/repository/mod.rs:4]: Can you use the same syntax as other dependencies, i.e. only group `std::fs`  and `std::io` uses together?

**Review by aawsome — CHANGES_REQUESTED** [review]

File: src/repository/mod.rs [hunk 2]
```diff
@@ -109,6 +109,30 @@ pub(crate) fn parse_command<'a, E: ParseError<&'a str>>(
...
+        password.pop();
+    }
+
+    // Ctrl-U should remove the line in terminals
+    if password.contains('') {
+        password = match password.rfind('') {
+            Some(last_ctrl_u_index) => password[last_ctrl_u_index + 1..].to_string(),
+            None => password,
+        };
+    }
```
[the remark this answers is not in the document]
Comment (fa993) [reply src/repository/mod.rs]: Fixed this

**Comment by fa993** [comment]
@aawsome Take a look now, and lemme know if anything else is needed

**Comment by aawsome** [comment]
LGTM - can you please squash your commits?

**Comment by fa993** [comment]
> LGTM - can you please squash your commits?

@aawsome done

**Pull request merged**